Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -197,8 +197,8 @@ void updateLayersForInteractionRegions(CALayer *layer, RemoteLayerTreeHost& host
             }
 
             setInteractionRegion(interactionRegionLayer.get(), region);
-            configureLayerForInteractionRegion(interactionRegionLayer.get(), makeString("WKInteractionRegion-"_s, String::number(region.elementIdentifier.toUInt64())));
             [interactionRegionLayer setCornerRadius:std::max(region.borderRadius, minimumBorderRadius)];
+            configureLayerForInteractionRegion(interactionRegionLayer.get(), makeString("WKInteractionRegion-"_s, String::number(region.elementIdentifier.toUInt64())));
```

Comment:
This is reordering two lines of code. The change log comment claims the patch does something else.

By the way, the `makeString` here does not need to use `String::number`.

Reply:
Whoops! Should've clearly marked this PR as a draft :)
Will make the `makeString` change though!